client/me/help/main.jsx

```diff
@@ -73,32 +80,61 @@ const Help = React.createClass( {
 				<CompactCard className="help__support-link" href="https://support.wordpress.com/" target="__blank">
 					<div className="help__support-link-section">
 						<h2 className="help__support-link-title">{ this.translate( 'All support articles' ) }</h2>
-						<p className="help__support-link-content">{ this.translate( 'Looking to learn more about a feature? Our docs have all the details.' ) }</p>
+						<p className="help__support-link-content">
+							{ this.translate( 'Looking to learn more about a feature? Our docs have all the details.' ) }
+						</p>
 					</div>
 				</CompactCard>
 				<CompactCard className="help__support-link" href="https://dailypost.wordpress.com/" target="__blank">
 					<div className="help__support-link-section">
 						<h2 className="help__support-link-title">{ this.translate( 'The Daily Post' ) }</h2>
-						<p className="help__support-link-content">{ this.translate( 'Get daily tips for your blog and connect with others to share your journey.' ) }</p>
+						<p className="help__support-link-content">
+							{ this.translate( 'Get daily tips for your blog and connect with others to share your journey.' ) }
+						</p>
 					</div>
 				</CompactCard>
 				<CompactCard className="help__support-link help__support-link-contact" href="/help/contact/">
 					<div className="help__support-link-section">
 						<h2 className="help__support-link-title">{ this.translate( 'Get in touch' ) }</h2>
-						<p className="help__support-link-content">{ this.translate( 'Can\'t find the answer? Drop us a line and we\'ll lend a hand.' ) }</p>
+						<p className="help__support-link-content">
+							{ this.translate( 'Can\'t find the answer? Drop us a line and we\'ll lend a hand.' ) }
+						</p>
 					</div>
 					<Button className="help__support-link-button" primary>{ this.translate( 'Contact Us' ) }</Button>
 				</CompactCard>
 		</div>
 		);
 	},
 
+	getCoursesTeaser: function() {
+		return (
+			<div className="help__course-teaser">
+				<Card href="#/help/courses">
+					<Gridicon className="help__course-teaser-icon" icon="help" size={ 30 } />
+					<div>
+						<span className="help__course-teaser-title">
+							{ this.translate( 'Courses' ) }
+						</span>
+						<span className="help__course-teaser-description" >
+							{ this.translate( 'Learn how to make the most of your site with these courses and webinars' ) }
+						</span>
+					</div>
+				</Card>
+			</div>
+		);
+	},
```

Comment:
This file in general can be restructured a bit so that the functions: `getCoursesTeaser`, `getSupportLinks` and `getHelpfulArticles` are in their own components. I'd save that for another pull request